backend/endpoints/datasets/bundle.sh

```diff
@@ -0,0 +1,9 @@
+pip install --requirement=datasets/requirements.txt --target=/asset-output \
+&& \
+mkdir --parents /asset-output/endpoints/datasets \
+&& \
+touch {/asset-output/endpoints/__init__.py,/asset-output/endpoints/datasets/__init__.py} \
```

Comment:
The brace expansion expands things like `prefix-{foo,bar}-suffix` to multiple arguments `prefix-foo-suffix prefix-bar-suffix`. Since there's no prefix or suffix around this brace expansion it is redundant, and the command can be simply `touch /asset-output/endpoints/__init__.py /asset-output/endpoints/datasets/__init__.py`.

Reply:
Fixed in 01730f4e95b2905ac868c2fb1b09f515c10e9871